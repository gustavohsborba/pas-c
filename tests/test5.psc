var

 j, k is int;
 a, j real;

 begin
 read(j);
 read(K);

 if (k <> 0)
 result := j/k
 else
 result := 0
 end;

 out(result);
 end