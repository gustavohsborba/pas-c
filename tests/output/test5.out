Opening file tests/test5.psc...
ERROR in line 4, column 11: Sintax error found - Expected TOK_IS,  but 'real' found
